internal/rest/dequeue_nodes.go

```diff
@@ -289,6 +296,11 @@ func (rest *RestOperations) CheckAndPublishForRetry(err error, publishKey avicac
 					rest.PublishKeyToSlowRetryLayer(publishKey, key)
 					return true
 				}
+				if strings.Contains(key, "StringGroup") && strings.Contains(*aviError.Message, lib.StringGroupCannotDeleteObjectError) {
```

Comment:
It is good that we are handling this error condition. But are we always get this error during deletion of string group when deleteConfig flag is set true? If so, this indicates , stringgroup deletion process is not kicked in after all VS deletions. So we should attempt string group deletion when all VS are deleted. This will help us in avoiding retries.